core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +60,30 @@ def map(exploration_model):
     def reduce(exp_id, list_of_exps):
         pass
 
+class ExpSummariesContributorsOneOffJob(jobs.BaseMapReduceJobManager):
```

Comment:
Will fix the nit. I wasn't aware of how to do manual testing for a job-- I was just running it using the test suite. How do I manually test/look at the admin page?